src/ext-test/java/org/opentripplanner/ext/vectortiles/VectorTilesResourceTest.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.ext.vectortiles;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.glassfish.grizzly.http.server.Request;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.opentripplanner.TestServerContext;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.test.support.HttpForTest;
+import org.opentripplanner.transit.service.TransitModel;
+
+class VectorTilesResourceTest {
+
+  @Test
+  void tileJson() {
```

Comment:
Btw a general question, should we prefix test methods with `test`, so for example `testTileJson`. I think this was discussed in some old pr between @leonardehrenfried and @vpaturet maybe but I'm not 100% sure what was the conclusion.

Reply:
The general Java guide is to start a method with a verb that describes the action. I generally agree with that.

On the other hand, this would mean that every test starts with the word "test". I think is a bit silly since no-one is going to call the method and the only place you see them is in the test report where it's clear it's a test. 

I don't think @vpaturet and I came to a conclusion. For me it's an aesthetic preference and not a deeply held believe so I'm happy to go along with the group decision.